python/gtsam/tests/test_Cal3Unified.py

```diff
@@ -23,6 +23,89 @@ def test_Cal3Unified(self):
         self.assertEqual(K.fx(), 1.)
         self.assertEqual(K.fx(), 1.)
 
+    def test_distortion(self):
+        "Stereographic fisheye model of focal length f, defined as r/f = 2*tan(theta/2)"
+        fx, fy, s, u0, v0 = 2, 2, 0, 0, 0
+        k1, k2, p1, p2 = 0, 0, 0, 0
+        xi = 1
+        stereographic = gtsam.Cal3Unified(fx, fy, s, u0, v0, k1, k2, p1, p2, xi)
+        x, y, z = 1, 0, 1
+        u, v = stereographic.uncalibrate([x, y])
+        r = np.linalg.norm([x, y, z])
+        # Note: 2*tan(atan2(x, z)/2) = 2/(1+sqrt(x^2+z^2))
+        self.assertAlmostEqual(2*np.tan(np.arctan2(x, z)/2), 2/(1+r))
+        self.assertAlmostEqual(u, 2/(1+r))
+        x2, y2 = stereographic.calibrate([u, v])
+        self.assertAlmostEqual(x2, x)
```

Comment:
Any reason these are `AlmostEqual` and not `Equal` (since tolerance is not specified)?